hydrus/itemhelpers.py

```diff
@@ -155,3 +164,160 @@ def items_delete_check_support(id_, class_type, path):
         error = e.get_HTTP()
         return set_response_headers(jsonify(error.generate()),
                                     status_code=error.code)
+
+
+def itemsCollection_get_support(collection, class_path, path, search_params):
```

Comment:
These functions are not really checking. They perform CRUD operations. The docstrings need to be updated.